test/appsignal/span_test.exs

```diff
@@ -180,16 +180,14 @@ defmodule AppsignalSpanTest do
   end
 
   describe ".set_name/2, with a span that doesn't have a reference" do
-    setup do
-      [return: Span.set_name(%Span{}, "test")]
-    end
+    setup :create_root_span
```

Comment:
If this is using the output of `:create_root_span` as the original span, is it still testing the case where the span passed is a span that has no reference, as the description of the test case says?